pkgs/servers/search/sphinxsearch/default.nix

```diff
@@ -13,12 +13,17 @@ stdenv.mkDerivation {
 
   configureFlags = [
     "--program-prefix=sphinxsearch-"
-    "--without-mysql"
+    "--with-mysql=${mariadb}/"
+    "--with-libexpat"
     "--enable-id64"
   ];
 
   nativeBuildInputs = [
-    pkgconfig
+    pkgconfig expat
```

Comment:
If you link with expat, then it should go to `buildInputs`?

Reply:
I'm not an expert, so it's up to you.

Reply:
Fixed.